templates/cloudflare-pages/package.json

```diff
@@ -6,7 +6,7 @@
     "dev:remix": "remix watch",
     "dev:wrangler": "cross-env NODE_ENV=development wrangler pages dev ./public",
     "dev": "remix build && run-p dev:*",
-    "start": "cross-env NODE_ENV=production npm run dev:wrangler"
+    "start": "cross-env NODE_ENV=production wrangler pages dev ./public"
```

Comment:
It seems weird to call `dev` when we're running a production build.

Does `wrangler` have a production command?

Reply:
> It seems weird to call dev when we're running a production build.

Sure 😅 

It appears that wrangler pages does not have a production command.

```
$ wrangler pages -h
wrangler pages

⚡️ Configure Cloudflare Pages

Commands:
  wrangler pages dev [directory] [-- command]  🧑‍💻   Develop your full-stack Pages application locally
  wrangler pages project                       ⚡️ Interact with your Pages projects
  wrangler pages deployment                    🚀 Interact with the deployments of a project
  wrangler pages publish [directory]           🆙 Publish a directory of static assets as a Pages deployment

Flags:
  -c, --config   Path to .toml configuration file  [string]
  -h, --help     Show help  [boolean]
  -v, --version  Show version number  [boolean]

🚧 'wrangler pages <command>' is a beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose
```

Is the start command unnecessary?

Reply:
I think @jacob-ebey could answer these wrangler-specific questions best.